app/controllers/api/v1/booking_requests_controller.rb

```diff
@@ -6,14 +6,22 @@ class BookingRequestsController < ActionController::Base
       before_action :authorise_api_user!
 
       def create
-        booking_request = BookingRequest.create!(booking_request_params)
-        BookingRequests.customer(booking_request).deliver_later
+        booking_request = BookingRequest.new(booking_request_params)
 
-        head :created
+        if booking_request.save
+          BookingRequests.customer(booking_request).deliver_later
+          head :created
+        else
+          render_errors(booking_request)
+        end
       end
 
       private
 
+      def render_errors(booking_request)
+        render json: { errors: booking_request.errors }, status: :unprocessable_entity
```

Comment:
Does this correctly format slot validations or do we just get `slot is invalid`?

Does this group the `slots` priority validation from the `booking request` object with the validations from the `slot` object?

Do we care (i.e. are we planning to use these validation messages anywhere)?